src/mainboard/aaeon/upsquared/fixed-dtfs.dts

```diff
@@ -6,9 +6,9 @@
 
     flash-info {
         compatible = "ore-flashinfo";
-        board-name = "UP Xtreme";
+        board-name = "UP Sqared";
```

Comment:
```suggestion
        board-name = "UP Squared";
```

Reply:
i fix this so it wont have duplicate name with the upxtreme board. Is this fine with you?